src/NuGet.Services.EndToEnd/Support/Package.cs

```diff
@@ -69,31 +70,7 @@ public static Package Create(PackageCreationContext context)
                 nupkgBytes = Array.AsReadOnly(bufferStream.ToArray());
             }
 
-            return new Package(context.Id, context.NormalizedVersion, context.FullVersion, nupkgBytes);
-        }
-
-        public static Package Create(PackageCreationContext context, IEnumerable<string> files)
-        {
-            return Create(context, files, new PackageProperties());
-        }
-
-        public static Package Create(PackageCreationContext context, IEnumerable<string> files, PackageProperties properties)
-        {
-            var physicalPackageFilesList = files
-                .Select(x => new { FileName = Path.GetFileName(x), Stream = GetMemoryStreamForFile(x) })
-                .Select(x => new PhysicalPackageFile(x.Stream) {
-                    TargetPath = $"lib/{x.FileName}"
-                });
-
-            ReadOnlyCollection<byte> nupkgBytes;
-            using (var nupkgStream = TestData.BuildPackageStreamForFiles(context, physicalPackageFilesList, properties.IsSymbolsPackage))
-            using(var bufferStream = new MemoryStream())
-            {
-                nupkgStream.CopyTo(bufferStream);
-                nupkgBytes = Array.AsReadOnly(bufferStream.ToArray());
-            }
-
-            return new Package(context.Id, context.NormalizedVersion, context.FullVersion, nupkgBytes, properties);
+            return new Package(context.Id, context.NormalizedVersion, context.FullVersion, nupkgBytes, context.Properties == null ? new PackageProperties() : context.Properties);
```

Comment:
nit: move the parameters to new line each for readability.

Reply:
Done